creator-cards/aleksei-zagoskin.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Aleksei Zagoskin",
+  "type": "CREATOR",
+  "tags": ".NET, C#, ASP.NET, Microservices, Architecture, REST API",
+  "github": "https://github.com/Powerz",
+  "channels": [
+    {
+      "name": "Blog",
+      "url": "https://zagosk.in"
+    }
+  ],
+  "feeds": [{
+    "type": "LinkTagRssType",
```

Comment:
hey @Powerz this has to be either "dotnet" or "personal" if you only talk about .NET (and azure etc) make it dotnet. If your feed has other content as well make it "personal"